test/kubemark/pre-existing/README.md

```diff
@@ -0,0 +1,45 @@
+# Kubemark Pre-existing Provider Guide
+
+**Pre-existing Kubemark Master**
```

Comment:
>So in total we have:
>
> 1 kubernetes master VM
   2 a bunch of node VMs
   3 kubemark master VM
>
> Just to ensure - you're going to reuse only 1 and 2 (together meaning kubernetes cluster), but not 3, right?
>
Yes, reuse only 1 and 2 (kubernetes cluster) and DIY (do it yourself) for 3 (kubemark master).